components/spiffs/spiffsgen.py

```diff
@@ -488,6 +488,11 @@ def main():
                         action="store_true",
                         default=True)
 
+    parser.add_argument("--follow-symlinks",
+                        help="For creating partition image, symbolic links are also taken into account. Specify if CONFIG_SPIFFS_FOLLOW_SYMLINKS.",
```

Comment:
"For creating partition image, symbolic links are also taken into account" -> "Take into account symbolic links during partition image creation"

The note "Specify if CONFIG_SPIFFS_FOLLOW_SYMLINKS" is not necessary here since it is not a config option used as a SPIFFS parameter in the source code.